### ISSUE TITLE
[¬Re] Setting Inventory Levels in a Bike Sharing Network

### ISSUE THREAD

**Issue by BarzolaT** [ReScience/submissions#68]
**Original article:** Setting Inventory Levels in a Bike Sharing Network

**PDF URL:** https://espaces-collaboratifs.grenet.fr/share/s/MyrKJU3BSbO1_LHBWDWD8g
**Metadata URL:** https://gitlab.com/BarzoThom/redatner/-/blob/main/Report/metadata.yaml
**Code URL:** https://gitlab.com/BarzoThom/redatner.git

**Scientific domain:** Operation Research
**Programming language:** Python
**Suggested editor:**

**Comment by rougier** [comment]
Very sorry for late answer. We'll assign an editor soon.

By the way, it seems one author missing on the pdf (list of authors ends with "and")

**Comment by rougier** [comment]
@khinsen @benoit-girard @oliviaguest Could any of you handle this replication (that has failed) in computer science ?

**Comment by khinsen** [comment]
@rougier I understand this well enough to handle this submission as an editor, but I am too far from the field to find suitable reviewers.

@BarzolaT Can you propose a few potential reviewers?

**Comment by khinsen** [comment]
@BarzolaT Is the author list of the submitted paper correct? It ends with "and", which looks suspicious.

**Comment by BarzolaT** [comment]
@khinsen Sorry for the late answer, do the reviewers have to be academic researchers? We are considering [Etienne Côme](https://www.comeetie.fr/) at IFSTTAR who worked on Bike Sharing Systems,  Vincent Leblond from [Tellae](https://tellae.fr/#/company) specialized in mobility Simulation, [Ariel Waserhole](https://www.linkedin.com/in/ariel-waserhole/) from Volterres who made his PhD on the same field.

Concerning the authors, we checked that the four authors appear correcly. Yes, an "and" appears  when using the rescience document class. But, we can't find the mistake in the metadata file. Sorry for not having reported that bug in the first place.

**Comment by khinsen** [comment]
Thanks @BarzolaT for the reviewer recommendations. We have no requirement about academic affiliations.

As for the author list, I tried to look at your metadata but failed. The URL you provide resolves to a page in a content management system, for a file "metadata.yaml" that I cannot see but download. Fine... except that downloading fails as well, with the error message "11076004 Web Script format 'yaml' is not registered". We will need your metadata file for publication, so please fix this!

The Code URL you show does not work either, the repository seems to be private. Please make it public!

**Comment by BarzolaT** [comment]
I am sorry for the inconvenience. We are facing some informatics issues at the university that might have caused the problem. I changed the repository to public access and modified the metadata link.

**Comment by khinsen** [comment]
Thanks @BarzolaT, I can access both items now!

**Comment by khinsen** [comment]
@comeetie Would be interested in reviewing this submission to ReScience C? Don't worry if you don't know anything about ReScience C so far - I will guide you along our somewhat unusual process.

**Comment by comeetie** [comment]
@khinsen Yes it's possible. I know a little bit this work since i will be in the jury of the PhD defense of Thomas Barzola. I hope this is not a problem ? And also sorry for the delay, the request ended up in my personal mail my work e-mail is etienne.come@univ-eiffel.fr

**Comment by khinsen** [comment]
@comeetie Thanks for replying, and even more fore accepting the task! I don't see a conflict in your being a member of a PhD jury on the same topic. You are on the side of the reviewers in both cases.

**Comment by khinsen** [comment]
@BarzolaT I haven't been able to locate the other two reviewers you suggested. The links you provide confirm their existence, but I cannot find e-mail addresses (or GitHub handles!), nor any CV-like information for judging if their past experience is relevant for this review. Can you help out? If you have their e-mail but don't want to post them here, please contact me by mail (my address is first-name [dot] last-name [at] cnrs [dot] fr).

**Comment by khinsen** [comment]
Also trying reviewer recruitment on Mastodon: https://scholar.social/@khinsen/109745104338743326

**Comment by khinsen** [comment]
@jeremyomer Would be interested in reviewing this submission to ReScience C? Don't worry if you don't know anything about ReScience C so far - I will guide you along our somewhat unusual process.

**Comment by khinsen** [comment]
@comeetie Did you make any progress with the review? If you run into any difficulties, don't hesitate to ask for help!

**Comment by ngast** [comment]
@khinsen @BarzolaT  Am I right by assuming that the reviewing process is stuck for now. Is there anything to do to make progress?

**Comment by khinsen** [comment]
@ngast Unfortunately your impression is correct. We need reviews to proceed, and for that we need reviewers. Any suggestions are welcome!

**Comment by rougier** [comment]
@ReScience/reviewers Help needed here to review this paper: https://github.com/ReScience/submissions/issues/68
